spec/support/m2bf2_xquery_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# helper methods for rspec
```

Comment:
Ok, I renamed the files. I also changed the `rspec --require` file names to just (e.g.) m2bf2.rb, so a command line to run the tests against m2bf2 would now be `rspec spec/titles/responsibilityStatement_from_245_246_spec.rb --tag bf2 -r m2bf2.rb`.

I am going to submit a separate PR to update the README with all this info..